GTNH Version History fails to parse

### Describe the problem

When trying to run the container under the `GTNH` type, regardless of the pack version specified, it will fail to start due to a jq error.

I've tried every container tag down to `2026.4.0-java25` up to the latest, but they all result in errors (some slightly different but none succeed).

This is very confusing as I created this container on May 4th and it downloaded fine, yet running the tagged version released on May 4th it fails to start even with the version I originally downloaded.

### Container definition

```
services:
  mc:
    image: itzg/minecraft-server:java25
    tty: true
    stdin_open: true
    ports:
      - 25567:25565
    environment:
      EULA: "TRUE"
      TYPE: GTNH
      GTNH_PACK_VERSION: 2.9.0-beta-2
      MODS: |
        "https://github.com/LazyFlesh/Garden-Of-Grind/releases/download/1.0.03/gogmod-1.0.03.jar"
      DIFFICULTY: hard
      SPAWN_PROTECTION: 0
      MEMORY: 8192M
      TZ: America/Chicago
      WHITELIST: |
        RyzechDev
      OPS: |
        RyzechDev
      DUMP_SERVER_PROPERTIES: "TRUE"
      CREATE_CONSOLE_IN_PIPE: "true"
      RCON_CMDS_STARTUP: gamerule doDaylightCycle false
      RCON_CMDS_FIRST_CONNECT: gamerule doDaylightCycle true
      RCON_CMDS_LAST_DISCONNECT: gamerule doDaylightCycle false
    volumes:
      - ./data:/data
networks: {}
```


### Container logs

```
mc-1  | ++ mc-image-helper java-release
mc-1  | + ((  25 >= 17  ))
mc-1  | + MC_IMAGE_HELPER_OPTS+=' --enable-native-access=ALL-UNNAMED'
mc-1  | + export MC_IMAGE_HELPER_OPTS
mc-1  | + export HOME=/data
mc-1  | + HOME=/data
mc-1  | ++ id -u
mc-1  | ++ id -g
mc-1  | ++ ls -lnd /data
mc-1  | + log 'Running as uid=1000 gid=1000 with /data as '\''drwxr-xr-x 17 1000 1000 4096 Jul  5 15:15 /data'\'''
mc-1  | + local oldState
mc-1  | ++ shopt -po xtrace
mc-1  | + oldState='set -o xtrace'
mc-1  | + shopt -u -o xtrace
mc-1  | [init] 2026-07-05 16:35:27-05:00 Running as uid=1000 gid=1000 with /data as 'drwxr-xr-x 17 1000 1000 4096 Jul  5 15:15 /data'
mc-1  | ++ paste -d, -s /etc/image.properties
mc-1  | + log 'Image info: buildtime=2026-07-05T12:43:55.619Z,version=java25,revision=d9ee3ee68ce20917b19cbd108dfb11b76dcad251'
mc-1  | + local oldState
mc-1  | ++ shopt -po xtrace
mc-1  | + oldState='set -o xtrace'
mc-1  | + shopt -u -o xtrace
mc-1  | [init] 2026-07-05 16:35:27-05:00 Image info: buildtime=2026-07-05T12:43:55.619Z,version=java25,revision=d9ee3ee68ce20917b19cbd108dfb11b76dcad251
mc-1  | + '[' '!' -e /data/eula.txt ']'
mc-1  | + isTrue false
mc-1  | + case "${1,,}" in
mc-1  | + return 1
mc-1  | + isTrue true
mc-1  | + case "${1,,}" in
mc-1  | + return 0
mc-1  | + set +x
mc-1  | + isTrue false
mc-1  | + case "${1,,}" in
mc-1  | + return 1
mc-1  | + proxyArgs=()
mc-1  | + [[ -n '' ]]
mc-1  | + proxyArgs=()
mc-1  | + addToProxyArgs http.proxyHost ''
mc-1  | + [[ -n '' ]]
mc-1  | + addToProxyArgs http.proxyPort ''
mc-1  | + [[ -n '' ]]
mc-1  | + addToProxyArgs http.nonProxyHosts ''
mc-1  | + [[ -n '' ]]
mc-1  | + LOCAL_IFS=$'\n\t'
mc-1  | + IFS=' '
mc-1  | + MC_IMAGE_HELPER_OPTS+=' '
mc-1  | + IFS=$'\n\t'
mc-1  | + export MC_IMAGE_HELPER_OPTS
mc-1  | + fixJavaPath
mc-1  | + which java
mc-1  | + cd /data
mc-1  | + isTrue false
mc-1  | + case "${1,,}" in
mc-1  | + return 1
mc-1  | + [[ -n '' ]]
mc-1  | + [[ -n '' ]]
mc-1  | + export DECLARED_TYPE=GTNH
mc-1  | + DECLARED_TYPE=GTNH
mc-1  | + export DECLARED_VERSION=LATEST
mc-1  | + DECLARED_VERSION=LATEST
mc-1  | + isTrue ''
mc-1  | + case "${1,,}" in
mc-1  | + return 1
mc-1  | + isTrue false
mc-1  | + case "${1,,}" in
mc-1  | + return 1
mc-1  | + isTrue false
mc-1  | + case "${1,,}" in
mc-1  | + return 1
mc-1  | + [[ -n gamerule doDaylightCycle false ]]
mc-1  | + log 'Starting RCON commands'
mc-1  | + local oldState
mc-1  | ++ shopt -po xtrace
mc-1  | + oldState='set -o xtrace'
mc-1  | + shopt -u -o xtrace
mc-1  | [init] 2026-07-05 16:35:27-05:00 Starting RCON commands
mc-1  | ++ dirname /image/scripts/start-configuration
mc-1  | + /image/scripts/start-rconcmds
mc-1  | [init] 2026-07-05 16:35:27-05:00 Rcon cmds functionality enabled
mc-1  | + isNumericElseSetToDefault RCON_CMDS_PERIOD 10
mc-1  | + local var_name=RCON_CMDS_PERIOD
mc-1  | + local default_value=10
mc-1  | + isNumeric 10
mc-1  | + [[ 10 =~ ^[0-9]+$ ]]
mc-1  | + checkIfNotZeroElseSetToDefault RCON_CMDS_PERIOD 10
mc-1  | + local var_name=RCON_CMDS_PERIOD
mc-1  | + local default_value=10
mc-1  | + '[' 10 -eq 0 ']'
mc-1  | + : ''
mc-1  | + [[ -n '' ]]
mc-1  | + case "${TYPE^^}" in
mc-1  | + MODPACK_PLATFORM=GTNH
mc-1  | + [[ -n GTNH ]]
mc-1  | + case "${MODPACK_PLATFORM^^}" in
mc-1  | ++ dirname /image/scripts/start-rconcmds
mc-1  | ++ dirname /image/scripts/start-configuration
mc-1  | + /image/scripts/auto/rcon-cmds-daemon.sh
mc-1  | + exec /image/scripts/start-deployGTNH
mc-1  | [init] 2026-07-05 16:35:27-05:00 Applying GTNH optimized server defaults
mc-1  | [init] 2026-07-05 16:35:27-05:00 DEBUG: Set MOTD=Greg Tech New Horizons 2.9.0-beta-2, ENABLE_COMMAND_BLOCK=true, DIFFICULTY=hard, LEVEL_TYPE=rwg, ALLOW_FLIGHT=true
mc-1  | + ensureRemoveAllModsOff MODPACK_PLATFORM=GTNH
mc-1  | + reason=MODPACK_PLATFORM=GTNH
mc-1  | + isTrue false
mc-1  | + case "${1,,}" in
mc-1  | + return 1
mc-1  | ++ mc-image-helper java-release
mc-1  | + java_version=25
mc-1  | + ((  java_version == 8  ))
mc-1  | + ((  java_version >= 17  ))
mc-1  | + export SERVER=/data/lwjgl3ify-forgePatches.jar
mc-1  | + SERVER=/data/lwjgl3ify-forgePatches.jar
mc-1  | + log 'TYPE=GTNH, setting Minecraft version to 1.7.10'
mc-1  | + local oldState
mc-1  | ++ shopt -po xtrace
mc-1  | + oldState='set -o xtrace'
mc-1  | + shopt -u -o xtrace
mc-1  | [init] 2026-07-05 16:35:28-05:00 TYPE=GTNH, setting Minecraft version to 1.7.10
mc-1  | + export VERSION=1.7.10
mc-1  | + VERSION=1.7.10
mc-1  | + handleGTNH
mc-1  | + : 2.9.0-beta-2
mc-1  | + : false
mc-1  | + : false
mc-1  | + debug 'GTNH VAR CHECK: GTNH_DELETE_BACKUPS=false, GTNH_PACK_VERSION=2.9.0-beta-2, TYPE=GTNH, SKIP_GTNH_UPDATE_CHECK=false'
mc-1  | + isDebugging
mc-1  | + isTrue true
mc-1  | + case "${1,,}" in
mc-1  | + return 0
mc-1  | + log 'DEBUG: GTNH VAR CHECK: GTNH_DELETE_BACKUPS=false, GTNH_PACK_VERSION=2.9.0-beta-2, TYPE=GTNH, SKIP_GTNH_UPDATE_CHECK=false'
mc-1  | + local oldState
mc-1  | ++ shopt -po xtrace
mc-1  | + oldState='set -o xtrace'
mc-1  | + shopt -u -o xtrace
mc-1  | [init] 2026-07-05 16:35:28-05:00 DEBUG: GTNH VAR CHECK: GTNH_DELETE_BACKUPS=false, GTNH_PACK_VERSION=2.9.0-beta-2, TYPE=GTNH, SKIP_GTNH_UPDATE_CHECK=false
mc-1  | + isTrue false
mc-1  | + case "${1,,}" in
mc-1  | + return 1
mc-1  | + [[ -n 2.9.0-beta-2 ]]
mc-1  | + isFalse false
mc-1  | + case "${1,,}" in
mc-1  | + return 0
mc-1  | + getGTNHdownloadPath
mc-1  | + gtnh_download_path=
mc-1  | + local release_object=
mc-1  | + local current_java_version=
mc-1  | ++ mc-image-helper java-release
mc-1  | + current_java_version=25
mc-1  | + '[' 2.9.0-beta-2 == latest-dev ']'
mc-1  | + '[' 2.9.0-beta-2 == latest ']'
mc-1  | ++ curl -fsSL https://downloads.gtnewhorizons.com/versions.json
mc-1  | ++ jq -r --arg USRIN 2.9.0-beta-2 '.versions|to_entries|sort_by(.value.releaseDate)|map(select(.key==$USRIN))|.[]'
mc-1  | jq: error (at <stdin>:630): null (null) has no keys
mc-1  | + release_object=
mc-1  | + logError 'Failed to retrieve release from https://downloads.gtnewhorizons.com/versions.json'
mc-1  | + isDebugging
mc-1  | + isTrue true
mc-1  | + case "${1,,}" in
mc-1  | + return 0
mc-1  | + set +x
mc-1  | [init] 2026-07-05 16:35:29-05:00 [ERROR] Failed to retrieve release from https://downloads.gtnewhorizons.com/versions.json 
mc-1  | + exit 1
mc-1 exited with code 1
```

--- a/scripts/start-deployGTNH
+++ b/scripts/start-deployGTNH
@@ -14,7 +14,7 @@ function getGTNHdownloadPath(){
   if [ "$GTNH_PACK_VERSION" == "latest-dev" ]; then
     if ! release_object="$(
       curl -fsSL "https://downloads.gtnewhorizons.com/versions.json" \
-        | jq -r '.versions|to_entries|sort_by(.value.releaseDate)|map(select(.value.title=="Beta release"))|.[-1]'
+        | jq -r '(.versions // .)|to_entries|sort_by(.value.releaseDate)|map(select(.value.title=="Beta release"))|.[-1]'
       )"; then logError "Failed to retrieve release from https://downloads.gtnewhorizons.com/versions.json"
       exit 1
     fi
@@ -23,7 +23,7 @@ function getGTNHdownloadPath(){
   elif [ "$GTNH_PACK_VERSION" == "latest" ]; then
     if ! release_object="$(
       curl -fsSL "https://downloads.gtnewhorizons.com/versions.json" \
-        | jq -r '.versions|to_entries|sort_by(.value.releaseDate)|map(select(.value.title=="Stable release"))|.[-1]'
+        | jq -r '(.versions // .)|to_entries|sort_by(.value.releaseDate)|map(select(.value.title=="Stable release"))|.[-1]'
       )"; then logError "Failed to retrieve release from https://downloads.gtnewhorizons.com/versions.json"
       exit 1
     fi
@@ -32,7 +32,7 @@ function getGTNHdownloadPath(){
   else
     if ! release_object="$(
       curl -fsSL "https://downloads.gtnewhorizons.com/versions.json" \
-        | jq -r --arg USRIN "$GTNH_PACK_VERSION" '.versions|to_entries|sort_by(.value.releaseDate)|map(select(.key==$USRIN))|.[]'
+        | jq -r --arg USRIN "$GTNH_PACK_VERSION" '(.versions // .)|to_entries|sort_by(.value.releaseDate)|map(select(.key==$USRIN))|.[]'
       )"; then logError "Failed to retrieve release from https://downloads.gtnewhorizons.com/versions.json"
       exit 1
     fi

